src/helpers/isRTL/isRTL.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Is RTL helper
+ *
+ * Detects if a locale needs RTL direction in the browser
+ */
+
+import rightToLeftLocales from './RTL-locales';
+
+const isRTL = locale => {
+  // Locales will in some cases be formed like e.g. "ar-AR"
+  // But we only need the first part to check for RTL locales
+  const cleanLocale = locale.indexOf('-') >= 0 ? locale.split('-')[0] : locale;
+
+  return rightToLeftLocales.indexOf(cleanLocale) >= 0;
```

Comment:
Simple enough. Is there an advantage to maintaining our own function rather than relying on a package like [rtl-detect](https://www.npmjs.com/package/rtl-detect)?

Reply:
I looked at rtl-detect and actually got the array of locales from this package. I decided to create this simple helper because rtl-detect included extra code/functionality that we don't need.

To me, it doesn't really make a difference. What do you think? Should I import rtl-detect instead?

Reply:
`rtl-detect` is pretty tiny and has no dependencies - getLangDir might not be useful here, but I feel like we might want it by the time we're through with all of this i18n process. I don't see a terrible downside to just using it instead of maintaining that list ourselves.

Reply:
Alright – I'll use rtl-detect instead 👍